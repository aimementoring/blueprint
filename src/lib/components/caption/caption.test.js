import React from 'react';
import renderer from 'react-test-renderer';
import Caption from './caption';

describe('Caption', () => {
  it('renders properly', () => {
    const tree = renderer.create(<Caption>Default caption</Caption>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('also works when text is passed as a prop', () => {
    const tree = renderer.create(<Caption text="Default Caption" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
