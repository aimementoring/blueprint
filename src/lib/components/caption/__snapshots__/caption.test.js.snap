// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Caption also works when text is passed as a prop 1`] = `
<div
  className="theme-base"
>
  <p
    className="caption "
  >
    Default Caption
  </p>
</div>
`;

exports[`Caption renders properly 1`] = `
<div
  className="theme-base"
>
  <p
    className="caption "
  >
    Default caption
  </p>
</div>
`;
